update-motd.d/40-environment

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+winhome_path="$HOME/winhome"
+
+printf " Environment variables:\n"
+printf "  DISPLAY    %s\n" "$DISPLAY"
+printf " \n"
+printf " Network\n"
+ip -o -f inet addr show | awk '{printf "  IPv4 for %-20s %s\n",$2, $4}'  |cut -f1 -d/
+printf " \n"
+printf " Windows Home Location: "
+printf " %s" "$winhome_path"
+printf " \n"
+printf " The windows drives are mounted at:\n"
+wslpath -u 'C:\' | awk '{printf "  %s", $1}'
+printf " \n"
+printf " To open the current folder with windows explorer run: wslview ."
```

Comment:
To open the current folder with Windows Explorer, run

Reply:
Done